flutter defines a list of pointer kinds that can scroll the screen.
however, it does not bother recognizing the pointer kind on linux,
so every pointer is set to be recognized as mouse. effectively, touch
can't scroll anything. this workarounds the issue by making mouse
one of the "touch-like device types".

Bug: https://github.com/flutter/flutter/issues/63209

--- ./packages/flutter/lib/src/widgets/scroll_configuration.dart.orig
+++ ./packages/flutter/lib/src/widgets/scroll_configuration.dart
@@ -25,6 +25,7 @@
   // The VoiceAccess sends pointer events with unknown type when scrolling
   // scrollables.
   PointerDeviceKind.unknown,
+  PointerDeviceKind.mouse,
 };
 
 /// The default overscroll indicator applied on [TargetPlatform.android].
